GitUI/CommandsDialogs/RevisionFileTreeControl.Designer.cs

```diff
@@ -248,6 +250,14 @@ private void InitializeComponent()
             this.openWithToolStripMenuItem.Text = "Open working directory file with...";
             this.openWithToolStripMenuItem.Click += new System.EventHandler(this.openWithToolStripMenuItem_Click);
             // 
+            // toggleExecutePermissionsToolStripMenuItem
+            // 
+            this.toggleExecutePermissionsToolStripMenuItem.Image = global::GitUI.Properties.Images.EditFile;
+            this.toggleExecutePermissionsToolStripMenuItem.Name = "toggleExecutePermissionsToolStripMenuItem";
+            this.toggleExecutePermissionsToolStripMenuItem.Size = new System.Drawing.Size(325, 22);
+            this.toggleExecutePermissionsToolStripMenuItem.Text = "Toggle execute permission...";
```

Comment:
Suggest something that clarify that worktree (current directory file) is changed
Toggle working directory file executable permissions...

Reply:
No text after ... (that should globally be changed to ellipsis)